spec/spec_helper.rb

```diff
@@ -1,2 +1,5 @@
-$LOAD_PATH.unshift File.expand_path("../../lib", __FILE__)
-require "mixlib/install"
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
+require 'mixlib/install'
```

Comment:
I would not include :allthethings: here in spec_helper. Even though this makes writing specs a little bit more convenient, it sometimes masks require bugs. E.g. `mixlib/install' might not require `mixlib/artifact_info` even though it needs it.